compiler/resolution/implementForallIntents.cpp

```diff
@@ -2209,7 +2218,8 @@ static void setShadowVarFlagsNew(Symbol* ovar, ShadowVarSymbol* svar, IntentTag
     svar->addFlag(FLAG_CONST_DUE_TO_TASK_FORALL_INTENT);
 }
 
-static void processShadowVarsNew(ForallStmt* fs, BlockStmt* body, int& numShadowVars)
+// Todo: is now time to get rid of pruning?
```

Comment:
Just noting this TODO - is this future work or something you are asking for this PR?